lib/config/validation.ts

```diff
@@ -135,6 +146,17 @@ export async function validateConfig(
         message: `The "${key}" object can only be configured at the top level of a config but was found inside "${parentPath}"`,
       });
     }
+    if (!isGlobalConfig) {
```

Comment:
Couldn't the optionsGlobals be lazily loaded inside here?

Reply:
Yep it can be.